src/styles/_colorschema_dark.scss

```diff
@@ -0,0 +1,82 @@
+.alpheios-popup.alpheios-color_schema_dark_class,
+.alpheios-panel.alpheios-color_schema_dark_class {
+	background: $alpheios-color_schema_dark_background;
+	color: $alpheios-color_schema_dark_color;
+
+	.alpheios-popup__header-selection,
+	.alpheios-definition__text,
+	.alpheios-morph__lexemes {
+		color: $alpheios-color_schema_dark_color;
+	}
+	.alpheios-morph__linkedattr,
+	.alpheios-popup__providers-link {
+		color: $alpheios-color_schema_dark_link_color;
+	}
+	.uk-button-primary {
+		background-color: $alpheios-color_schema_dark_color;
+		color: $alpheios-toolbar-active-color;
+		font-weight: bold;
+	}
+	.uk-form-label,
+	.alpheios-info, 
+	.alpheios-info .alpheios-info__helptext p,
```

Comment:
I probably put it incorrectly. It's not that I don't like CSS class names that are long: this is necessary for them to be unique. I have a problem with selectors containing multiple nested elements such as `.alpheios-info .alpheios-info__helptext p`. This selector targets a `p` element inside a `.alpheios-info__helptext` element that is inside a `.alpheios-info` element. If an element `alpheios-info__helptext` is moved out of  `.alpheios-info`, a `p` will lose its styling. This is usually not what is desired and is hard to track. 

We will also end up with a `.alpheios-info .alpheios-info__helptext p` rule that will not be used anywhere, and that will be hard to track too. 

If we're using BEM, then `alpheios-info__helptext` should be a unique class name. So if we don't want to assign a special class to `p` elements (especially if there are several of them, assigning a class to a single element is perfectly fine), a shorter selector like `.alpheios-info__helptext p` should work and be enough. It is also safer as we can move `alpheios-info__helptext` element anywhere and it will keep its styling.

Sometimes we can use even `..alpheios-info__helptext`. It will be inherited by `p`s. So I think this shorter version is even better, if possible.

Long (multi-class) selectors leads to specificity wars, where a need to restyle something styled with `.b .c .d` leads to use a stronger `.a .b .c .d` selector and so forth.

That's what I meant. Did I explain my reasoning clear enough? What do you think?